test/functional/sqlsrv/MsCommon.inc

```diff
@@ -467,4 +454,13 @@ function handleErrors()
     }
 }
 
+// non-UTF8 locale support in ODBC 17 and above only
+function isLocaleSupported()
+{
```

Comment:
Please use a more generic name to check for ODBC 17... other tests that require this ODBC ver but have nothing to do with locale can use this method :)

Reply:
this is generic, checking for ODBC 17 is meaningless, why? it's because locale support is there.
the only other place we check for ODBC version is isQualified it's the same issue there we don't call the version out there either. (btw I think it should be isAEQualified but that's a diff topic...